client/src/navbar.js

```diff
@@ -0,0 +1,189 @@
+// version 3
+import React, { useState } from "react";
+import "./navbar.css";
+
+const Navbar = () => {
+  const [burger_class, setBurgerClass] = useState("burger-bar unclicked");
+  const [menu_class, setMenuClass] = useState("menu hidden");
+  const [isMenuClicked, setIsMenuClicked] = useState(false);
+
+  const updateMenu = () => {
+    if (!isMenuClicked) {
```

Comment:
It looks like these states all only ever get changed at the same time - could you think of a way to only have one state rather than three?